Makefile

```diff
@@ -3,29 +3,12 @@ GO = /usr/bin/env go
 GOFMT = /usr/bin/env gofmt
 VERSION ?= dev-$(shell date +%FT%H-%M-%S-%Z)
 
-BINARY ?= kubeapps
 GO_PACKAGES = ./...
 GO_FILES := $(shell find $(shell $(GO) list -f '{{.Dir}}' $(GO_PACKAGES)) -name \*.go)
-GO_FLAGS = -ldflags="-s -w -X github.com/kubeapps/kubeapps/cmd/kubeapps.VERSION=${VERSION}"
-EMBEDDED_STATIC = generated/statik/statik.go
 
-default: kubeapps
+default: test
```

Comment:
test? 

Maybe we want to make `default: all` so it builds? That's what I suggested as part of this https://github.com/kubeapps/kubeapps/issues/461#issue-349249529

Reply:
:+1: